### PR TITLE
add create view statement

### PR DESCRIPTION
**Pull request by pfent**

### CODE REVIEW

File: content/references/sqlreference/statements/createview.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: "Reference: Create View Statement"
+linkTitle: "Create View"
+weight: 14
+---
+
+You can use create view to define a *virtual* table, specified by a query.
+When you reference the view, CedarDB reruns the query as if you would have specified the table as a subselect.
```
Comment (lukasvogel) [inline content/references/sqlreference/statements/createview.md:8]: What is a subselect?
Maybe expand usage example: 

```
create view funny_movies (...) 
select * form funny_movies where director = 'X'

is equivalent to

select * from (...)
```

File: content/references/sqlreference/statements/createview.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Views are similar to common table expressions (CTEs), but are stored persistently in the schema.
```
Comment (lukasvogel) [inline content/references/sqlreference/statements/createview.md]: What does "stored persistently in the schema" mean? -> Survives connections and server restarts.

Comment (lukasvogel) [inline content/references/sqlreference/statements/createview.md]: Link to CTE page?

File: content/references/sqlreference/statements/createview.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Views are a good way to structure your SQL.
+Views help to encapsulate the structure of your data and allow decoupling an interface from the underlying tables.
+CedarDB does not treat views as an optimization barrier, but optimizes across the whole query.
+Therefore, querying data from a view has exactly the same performance as specifying the query manually.
+
+However, you should be mindful when querying data from views which contain joins.
+When the view contains joins between multiple tables, but you only reference one of them, the query might execute
+unnecessary joins. 
+When your schema enforces foreign keys, CedarDB's query optimizer can often eliminate joins with no effect.
```
Comment (lukasvogel) [inline content/references/sqlreference/statements/createview.md]: Why? How? Maybe an example would help.

"... with no effect" -> "can infer which joins have no effect on the query result and eliminate them"

**Review by lukasvogel — APPROVED** [review]

File: content/references/sqlreference/statements/createview.md (hunk 2 above)
Comment (pfent) [reply content/references/sqlreference/statements/createview.md]: Wir haben noch keine CTE page, PRs welcome :hugs:

**Pull request merged**